index.bs

```diff
@@ -313,19 +313,21 @@ The <dfn method for="StorageBucket">persist()</dfn> method steps are:
 
 1. Let |p| be [=a new promise=].
 
-1. Run the following steps [=in parallel=]:
+1. Run the following steps [=in parallel=]: 
 
-    1. If |bucket|'s [=storage bucket/removed=] flag is true, [=reject=] |p| with {{InvalidStateError}}.
+    1. If |bucket|'s [=storage bucket/removed=] flag is true, then [=queue a storage task=] to [=reject=] |p| with an {{InvalidStateError}}.
 
-    1. If |bucket|'s [=bucket mode=] is `"persistent"`, [=/resolve=] |p| with true.
+    1. Let |persisted| be true if |bucket|'s [=bucket mode=] is `"persistent"`.
 
     1. Otherwise,
 
         1. Let |permission| be the result of [=getting the current permission state=] with `"persistent-storage"` and |environment|.
 
-        1. If |permission| is "{{PermissionState/granted}}", then set |bucket|'s [=bucket mode=] to `"persistent"` and [=queue a storage task=] to [=/resolve=] |p| with true.
+        1. If |permission| is "{{PermissionState/granted}}", then set |bucket|'s [=bucket mode=] to `"persistent"` and set |persisted| to true.
+
+        1. Otherwise, set |persisted| to false.
 
-        1. Otherwise, [=/resolve=] |p| with false.
+    1. [=Queue a storage task=] to [=/resolve=] |p| with |persisted|
```

Comment:
nit: I was wondering why we use `[=/resolve=]` (with slash) but `[=reject=]` (without slash). I think both work in this case, but we should choose one and be consistent.

https://speced.github.io/bikeshed/#autolink-inside

Reply:
Good point, updated for the bucket interface. I'll follow-up for the bucket manager